jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby-1.7-vs-9k/Jenkinsfile

```diff
@@ -27,11 +30,14 @@ pipeline.multipass_pipeline([
         ],
         [
                 job_name: 'oss-latest-jruby-9k',
```

Comment:
I wouldn't be surprised if they would conflict, but I'm not sure. Updated the names either way